source/lib/unit-converters.ts

```diff
@@ -1,3 +1,6 @@
 export function lovelacesToAda(lovelaces: number) {
   return lovelaces / 1000000;
 }
+
+export const lovelacesStringToAdaNumber = (lovelaces: string) =>
```

Comment:
I've implemented a util in `cardano-js` that we can refactor this to, but good for now